example/view/transform_golden.txt

```diff
@@ -0,0 +1 @@
+[0.5,1,1.5]
```

Comment:
This is now redundant

Reply:
This file is included as a snippet in the docs: `/// \include example/view/transform_golden.txt`